.travis.yml

```diff
@@ -11,14 +11,14 @@ addons:
 install:
   - set -e
   - bin/setup_base
-  - $TEST_SKIP bin/compile_ovs
-  - $TEST_SKIP bin/build_docker ${DOCKER_BUILD:-pull}
+  - $SKIP_INSTALL bin/compile_ovs
+  - $SKIP_INSTALL bin/build_docker ${DOCKER_BUILD:-pull}
 script:
   - set -e
-  - $TEST_SKIP bin/setup_stack local
+  - $SKIP_STACK bin/setup_stack local $MODE
   - bin/run_test_set
 env:
   matrix:
     - TEST_SET=stack
     - TEST_SET=reconfig
-    - TEST_SET=topology TEST_SKIP=true
+    - TEST_SET=topology SKIP_INSTALL=true MODE=dva
```

Comment:
On second thought, I don't think 1) this will work, and 2) it's the right thing to do. First, you have to run the install stuff in order to setup the stack, so you can't do the SKIP_INSTALL part and then try to use the stack. Second, I think we should keep test_topology to be purely functional and not integration. So it can keep the things that are just static file comparisons, but anything that's more dynamic (requires a functioning stack) should go elsewhere.

So, we can either move the work you did here into one of the other test setups, or just create a new one (so there's four instead of three). I think creating a new one will likely be best (although slow)... it can be something like test_access or something (all about testing various dynamic access policies).

Reply:
Created a new test_set